README.md

```diff
@@ -4,18 +4,17 @@
 
 ## Local Development (docker) (recommended):
 
-`docker run --rm -it -p 8001:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:5.1.0`
+`docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material`
```

Comment:
I changed this to 8001 because `8000` conflicts with Kong port when we have it running.